### ISSUE TITLE
virtiofs.exe logging doesn't work when started as a Windows service

### ISSUE THREAD

**Issue by jwatt** [virtio-win/kvm-guest-drivers-windows#528]
I've tried running the following to add logging to the `Virtio-FS Service`:

```
sc config "Virtio-FS Service" binpath="\"%ProgramFiles%\Virtio-Win\VioFS\virtiofs.exe\" -d -1 -D %SystemDrive%\my-viofs.log"
```

Unfortunately, on restarting the service, the file `C:\my-viofs.log` is not created.

However, if I then stop the service, copy the `Path to executable` from the services `Properties` and run it manually in an `Administrator Command Prompt`, then `C:\my-viofs.log` is successfully created.

**Comment by menli820** [comment]
Thanks for bug report. @hammerg I also can reproduce this issue on downstream, Do I need to file a bug to track this issue ?

pkg:
qemu-kvm-5.2.0-3.module+el8.4.0+9499+42e58f08.x86_64
kernel-4.18.0-272.el8.x86_64
seabios-1.14.0-1.module+el8.4.0+8855+a9e237a9.x86_64
virtio-win-1.9.14-4.el8.iso

steps：
1. config  Virtio-FS Service to add log  with command:
sc config "Virtio-FS Service" binpath="\"%ProgramFiles%\Virtio-Win\VioFS\virtiofs.exe\" -d -1 -D C:\my-viofs.log"

2. restart Virtio-FS Service

Actual result：
after step2， no log created.

**Comment by hammerg** [comment]
> Thanks for bug report. @hammerg I also can reproduce this issue on downstream, Do I need to file a bug to track this issue ?

@menli820 Your call. I prefer a bug report as it is much easier to track. Thanks.

**Comment by menli820** [comment]
> > Thanks for bug report. @hammerg I also can reproduce this issue on downstream, Do I need to file a bug to track this issue ?
> 
> @menli820 Your call. I prefer a bug report as it is much easier to track. Thanks.

@hammerg Thanks for reply, can track with https://bugzilla.redhat.com/show_bug.cgi?id=1918312

**Comment by YanVugenfirer** [comment]
Solved by PR#690
